packages/react-router/__tests__/DataMemoryRouter-test.tsx

```diff
@@ -27,7 +27,9 @@ import {
   UNSAFE_DataRouterContext,
   MemoryRouter,
   Routes,
-} from "../index";
+} from "react-router";
```

Comment:
I updated tests to match the pattern already used elsewhere where unit tests import from the module name, and not from a relative path.  

I think there's room for larger discussion here in a follow up, but with module imports (as opposed to relative imports) we get a nicer distinction of public versus private APIs.  This should generally lead us towards testing public APIs and not implementation details, while still allowing us to grab internal private utilities as needed.

If we test the `dist/` directories, we have to publicly expose any type of `_resetModuleScope` so our tests can access it through the `dist/` directory.